crates/bevy_ui/src/focus.rs

```diff
@@ -89,8 +89,9 @@ pub fn ui_focus_system(
         mouse_button_input.just_pressed(MouseButton::Left) || touches_input.any_just_pressed();
 
     let cursor_position = windows
-        .get_primary()
-        .and_then(|window| window.cursor_position())
+        .iter()
+        .filter_map(|window| window.cursor_position())
+        .next()
```

Comment:
I think with my proposed change, it will end up looking like that
```rust
windows
  .get(ui_window_id)
  .and_then(|window| window.cursor_position())
```

Reply:
While I totally agree with you, we will need information on where the UI entity want to render.
I cannot find an easy way to tell our window which UI to render.

Reply:
This is why I suggest to assume unique UI camera for now. You won't get the issue I mentioned, even if you still can't handle multiple interactive UI windows.

Reply:
Changed to consider only currently focused window with camera.